editor/scene/sprite_frames_editor_plugin.cpp

```diff
@@ -1336,6 +1338,104 @@ void SpriteFramesEditor::_animation_speed_changed(double p_value) {
 	undo_redo->commit_action();
 }
 
+void SpriteFramesEditor::_animation_remove_undo_redo(const StringName &p_action_name, const Vector<ClipboardSpriteFrames::Frame> *p_frames) {
+	StringName new_edited = _find_next_animation();
+	int frame_count = frames->get_frame_count(edited_anim);
+	EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
+	undo_redo->create_action(TTR(p_action_name), UndoRedo::MERGE_DISABLE, frames.ptr());
```

Comment:
This translation macro won't work afaik, or at least it won't make it extracted, it should be called for the argument to `_animation_remove_undo_redo` instead 
```suggestion
	undo_redo->create_action(p_action_name, UndoRedo::MERGE_DISABLE, frames.ptr());
```